src/jit/importer.cpp

```diff
@@ -5188,25 +4485,32 @@ NamedIntrinsic Compiler::lookupNamedIntrinsic(CORINFO_METHOD_HANDLE method)
                                 result = NI_Base_Vector256_AsUInt64;
                             }
                         }
-                        else if (strcmp(methodName, "get_Zero") == 0)
+                        else if (strcmp(methodName, "CreateScalarUnsafe") == 0)
```

Comment:
I think we should unify the Vector intrinsic name lookup code with other hardware intrinsics that using the table-driven framework. The current big if-sequence seems not worthy for perf. And more important, we have discussed several times that we want to improve the intrinsic recognition perf, so a unified name lookup is desired.
But that could be a follow-up PR.

Reply:
Agreed. I'll submit a follow up PR once this one is merged.